fabric-biomes-v1/build.gradle

```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-biomes-v1"
-version = getSubprojectVersion(project, "0.1.5")
+version = getSubprojectVersion(project, "0.2.0")
```

Comment:
Is this still 0.2.0 or is it 0.3.0 now?

Reply:
Not sure. This targets the 1.15 branch, and your biomes pr targets 1.16-snapshot. I think I'll make it 0.2.0 here, and the 1.16 snapshot version can be bumped to 0.3.0 when it's merged

Reply:
alright